tests/test_standard_tags.py

```diff
@@ -132,7 +132,7 @@ def test_status_code(self):
             interfaces.library.add_span_tag_validation(request=r, tags=tags)
 
 
-@released(dotnet="2.13.0", golang="1.39.0", nodejs="2.11.0", php="?", python="?", ruby="?")
+@released(dotnet="2.13.0", golang="1.39.0", nodejs="2.11.0", php="?", python="1.6.0rc1.dev", ruby="?")
```

Comment:
The current version of the tracer in githug is `python@1.5.0rc2.dev49+g57a03ec5` -> https://github.com/DataDog/system-tests/actions/runs/3242665553/jobs/5316283397#step:9:43

It means that the test is not executed here.

Reply:
@christophe-papazian did you saw my comment ?